python/modules/BtagSFWeightProducer.py

```diff
@@ -151,7 +157,8 @@ def analyze(self, event):
                 eff = self.h_eff_udsg.GetBinContent(pt_bin, eta_bin);
             
             if self.FastSim:
-                btagSF = jet.btagSF*jet.btagSF_FS
+                btagSF = jet.btagSF
+                btagSF_FS=jet.btagSF_FS
```

Comment:
This is missing for fullsim, while the rest of the code call btagSF_FS even for fullsim. I will fix it